src/CondenserDotNet.Client/ConfigurationRegistry.cs

```diff
@@ -35,8 +45,14 @@ public Task<bool> AddStaticKeyPathAsync(string keyPath)
                 return -1;
             }
             var content = await response.Content.ReadAsStringAsync();
-            var keys = JsonConvert.DeserializeObject<KeyValue[]>(content);
-            var dictionary = keys.ToDictionary(kv => kv.Key.Substring(keyPath.Length).Replace('/', ':'), kv => kv.Value == null ? null : kv.ValueFromBase64(), StringComparer.OrdinalIgnoreCase);
+
```

Comment:
My fault from the original one but maybe we put these values (/ : ) as private static readonly at the top.